streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/FlinkClusterServiceImpl.java

```diff
@@ -429,13 +430,15 @@ private ShutDownResponse shutdownInternal(FlinkCluster flinkCluster, String clus
   }
 
   private DeployResponse deployInternal(FlinkCluster flinkCluster)
-      throws InterruptedException, ExecutionException, TimeoutException {
+      throws InterruptedException, ExecutionException, TimeoutException, JsonProcessingException {
```

Comment:
Emmm... I'm not sure which section of the code will throw an JsonProcessingException.